nautobot_design_builder/models.py

```diff
@@ -103,11 +103,12 @@ class Design(PrimaryModel):
     to a saved graphql query at some point in the future.
     """
 
-    # TODO: Add version field (future feature)
     # TODO: Add saved graphql query (future feature)
     # TODO: Add a template mapping to get custom payload (future feature)
     job = models.ForeignKey(to=JobModel, on_delete=models.PROTECT, editable=False)
-
+    version = models.CharField(max_length=20, default="0.0.0")
+    description = models.CharField(max_length=255, blank=True, default="")
+    docs = models.CharField(max_length=4096, blank=True, default="", editable=False)
```

Comment:
If these two fields are in the design job class why do we need to also store them in the database?

Reply:
they are not. The only field persisted by the DesignJob (or Job) is the description. The version and docs are in the model, but not persisted in the DB.